src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -337,10 +340,9 @@ const MediaPlayer = ({ enableFileDownload = false, enablePIP = false }) => {
     sources: isMultiSource
       ? playerConfig.sources[srcIndex]
       : playerConfig.sources,
-    tracks: playerConfig.tracks,
-    // Disable native text track functionality in Safari
+    // Enable native text track functionality in iPhones when not using HLS streams
     html5: {
-      nativeTextTracks: false
+      nativeTextTracks: IS_IOS && IS_IPHONE && !isStream
```

Comment:
Is this functionality something that needs to be applied to all iOS devices or just iPhones?

Reply:
Just iPhones.